app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
I agree that would be good too -- though unexpected defaults aren't necessarily great either.